package androidx.compose.foundation;

import androidx.compose.foundation.gestures.Orientation;
import androidx.compose.ui.f;
import androidx.compose.ui.graphics.a5;
import androidx.compose.ui.graphics.k4;
import androidx.compose.ui.unit.LayoutDirection;

/* loaded from: classes.dex */
public abstract class g {

    /* renamed from: a, reason: collision with root package name */
    private static final float f2700a = o0.i.g(30);

    /* renamed from: b, reason: collision with root package name */
    private static final androidx.compose.ui.f f2701b;

    /* renamed from: c, reason: collision with root package name */
    private static final androidx.compose.ui.f f2702c;

    /* loaded from: classes.dex */
    public static final class a implements a5 {
        a() {
        }

        @Override // androidx.compose.ui.graphics.a5
        public k4 a(long j11, LayoutDirection layoutDirection, o0.e eVar) {
            float c02 = eVar.c0(g.b());
            return new k4.b(new y.i(0.0f, -c02, y.m.i(j11), y.m.g(j11) + c02));
        }
    }

    /* loaded from: classes.dex */
    public static final class b implements a5 {
        b() {
        }

        @Override // androidx.compose.ui.graphics.a5
        public k4 a(long j11, LayoutDirection layoutDirection, o0.e eVar) {
            float c02 = eVar.c0(g.b());
            return new k4.b(new y.i(-c02, 0.0f, y.m.i(j11) + c02, y.m.g(j11)));
        }
    }

    static {
        f.a aVar = androidx.compose.ui.f.f4253a;
        f2701b = androidx.compose.ui.draw.d.a(aVar, new a());
        f2702c = androidx.compose.ui.draw.d.a(aVar, new b());
    }

    public static final androidx.compose.ui.f a(androidx.compose.ui.f fVar, Orientation orientation) {
        return fVar.e(orientation == Orientation.Vertical ? f2702c : f2701b);
    }

    public static final float b() {
        return f2700a;
    }
}
